hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -595,6 +595,34 @@ public final class OzoneConfigKeys {
       OZONE_CLIENT_BUCKET_REPLICATION_CONFIG_REFRESH_PERIOD_DEFAULT_MS =
       300 * 1000;
 
+  public static final String HDDS_DATANODE_DNS_INTERFACE_KEY =
```

Comment:
Our config organization isn't great, but these should probably be moved to:
- `HddsConfigKeys` if used by datanode and SCM
- `DatanodeConfiguration` if used only by datanode
- `ScmConfigKeys` if used only by SCM